pkg/apis/options/providers.go

```diff
@@ -252,10 +252,6 @@ func providerDefaults() Providers {
 				InsecureAllowUnverifiedEmail: false,
 				InsecureSkipNonce:            true,
 				SkipDiscovery:                false,
-				UserIDClaim:                  OIDCEmailClaim, // Deprecated: Use OIDCEmailClaim
-				EmailClaim:                   OIDCEmailClaim,
-				GroupsClaim:                  OIDCGroupsClaim,
-				AudienceClaims:               OIDCAudienceClaims,
```

Comment:
Why would you remove the default values here? This will most likely have impact on the keycloak-oidc provider

Reply:
`providerDefaults()` is used only in `options.NewOptions()`. Then, `opts.Providers[0]` is passed to `providers.NewProvider()`. Since `providers.NewProvider()` is a constructor and checks for default values, I deleted it to test whether the default settings work without problems.

If it's weird, I'll revert it.